### PR TITLE
Update to ReactiveUI 10

### PR DESCRIPTION
**Pull request by worldbeater**
**What kind of change does this PR introduce?**

This PR updates the ReactiveUI required package from `9.*` to `10.*` version range.

**What is the current behavior?**

Currently, ReactiveUI.Validation depends on an older version of ReactiveUI.

**What is the new behavior?**

Now, ReactiveUI.Validation depends on ReactiveUI 10. 

**What might this PR break?**

Attempted to resolve all of the breaking changes, but let's see what the CI checks will tell.

### CODE REVIEW

File: src/ReactiveUI.Validation/TemplateGenerators/PropertyValidationGenerator.cs [hunk 1]
```diff
@@ -36,22 +36,22 @@ public sealed class BasePropertyValidation<TViewModel, TProperty1, TProperty2>
...
         /// Represents the current value.
         /// </summary>
-        private readonly Subject<Tuple<TProperty1, TProperty2>> _valueSubject = new Subject<Tuple<TProperty1, TProperty2>>();
+        private readonly Subject<ValueTuple<TProperty1, TProperty2>> _valueSubject = new Subject<ValueTuple<TProperty1, TProperty2>>();
 
         /// <summary>
         /// The validation message factory.
         /// </summary>
-        private readonly Func<Tuple<TProperty1, TProperty2>, bool, ValidationText> _message;
+        private readonly Func<ValueTuple<TProperty1, TProperty2>, bool, ValidationText> _message;
```
Comment (glennawatson) [inline src/ReactiveUI.Validation/TemplateGenerators/PropertyValidationGenerator.cs]: Instead of `ValueTuple` do `(TProperty1 prop1, TProperty2 prop2)`

Comment (glennawatson) [reply src/ReactiveUI.Validation/TemplateGenerators/PropertyValidationGenerator.cs]: Or (TProperty1, TProperty2) if you just want to use Item1/Item2 etc

**Comment by glennawatson** [comment]
As part of RxUI 10 we forced users to have to upgrade to Android 9 btw, so might be worth setting that in the csproj.

**Comment by glennawatson** [comment]
Also just looks like the API validation is missing the value tuple changes and that's all that's preventing the compile.

**Comment by worldbeater** [comment]
Thanks for the comments Glenn, going to fix the mentioned issues tonight.

**Pull request merged**